docs/source/conf.py

```diff
@@ -0,0 +1,90 @@
+import pathlib
+import sys
+
```

Comment:
Why conf.py? I think we should follow the stucture used by msrpime etc using jupyterbook